addons/mail/static/tests/suggestion/suggestion.test.js

```diff
@@ -1340,3 +1340,43 @@ test("Mention with @-role trigger one RPC only", async () => {
         "/web/dataset/call_kw/res.partner/get_mention_suggestions_from_channel",
     ]);
 });
+
+test("Mentioning @everyone with more than 50 members shows warning dialog", async () => {
+    const pyEnv = await startServer();
+    const partnerIds = pyEnv["res.partner"].create(
+        Array(MENTION_AMOUNT_WARNING + 1).fill({ name: "" })
```

Comment:
I think we're trying to use `range` instead of array magic.

```suggestion
        range(0, MENTION_AMOUNT_WARNING + 1).map(() => ({ name: "" }))
```